vendor/popover_menu.js

```diff
@@ -0,0 +1,74 @@
++function ($) {
```

Comment:
should we stick this in a shared repo (ux-components one?) and have it be our first thing that we pull into both web and backdraft?

Otherwise how do we prevent these from drifting